### PR TITLE
More v1.5 updates, new DOI and change of license to BSD

### PR DESCRIPTION
**Pull request by langner**
As agreed earlier, changing license here from LGPL to BSD in order to facilitate compatibility with Avogadro.

### CODE REVIEW

File: THANKS [hunk 1]
```diff
@@ -1,45 +1,55 @@
-The developers of cclib would like the thank the following (in alphabetical order) who have contributed in some way to cclib:
+W would like the thank the following who have contributed in some way to cclib:
```
Comment (berquist) [inline THANKS]: typo

**Comment by berquist** [comment]
I'm fine with the license change.

File: THANKS (hunk 1 above)
Comment (langner) [inline THANKS]: Fixed.

File: THANKS [hunk 2]
```diff
@@ -52,4 +52,4 @@ Developers of cclib
 Eric Berquist
```
Comment (berquist) [inline THANKS]: Ah, might as well join the party: Eric J. Berquist

Comment (langner) [inline THANKS]: Done.

**Comment by baoilleach** [comment]
Sounds good to me.

**Comment by cstein** [comment]
As a contributor I am happy with this change. 👍

**Pull request merged**